docs/core/tools/dotnet-test.md

```diff
@@ -122,6 +122,12 @@ List all of the discovered tests in the current project.
 
 Sets the verbosity level of the command. Allowed values are `q[uiet]`, `m[inimal]`, `n[ormal]`, `d[etailed]`, and `diag[nostic]`.
```

Comment:
This needs some kind of syntactic summary on lines 21-22. Possibly something like `-- \<RUNSETTINGS_ARGUMENTS>`

Unrelated to your PR, it would also be good to break lines 21-22 into additional lines, since it's formatted badly in a scrolling window (see https://review.docs.microsoft.com/en-us/dotnet/core/tools/dotnet-test?branch=pr-en-us-8618&tabs=netcore21).